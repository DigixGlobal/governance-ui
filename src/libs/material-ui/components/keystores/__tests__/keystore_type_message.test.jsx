import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import KeystoreTypeMessage from '../keystore_type_message';

jest.mock('semantic-ui-react');

const keystoreType = {
  id: 1,
  name: 'keystore test 1',
  icon: 'testIcon',
  color: 'blue',
  description: 'test description',
};

describe('<KeystoreTypeMessage />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <KeystoreTypeMessage keystoreType={keystoreType} />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
