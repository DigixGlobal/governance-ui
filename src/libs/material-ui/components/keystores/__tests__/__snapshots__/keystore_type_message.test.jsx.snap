// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<KeystoreTypeMessage /> renders correctly 1`] = `
<Message
  icon={true}
  info={true}
>
  <Icon
    as="i"
    name="testIcon"
  />
  <MessageContent>
    <MessageHeader>
      keystore test 1
    </MessageHeader>
    test description
  </MessageContent>
</Message>
`;
